politeiawww/cmd/politeiawww_dbutil/config/config.go

```diff
@@ -0,0 +1,231 @@
+package config
+
+import (
+	"fmt"
+	"hash/fnv"
+	"os"
+	"os/user"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	flags "github.com/btcsuite/go-flags"
+	"github.com/decred/dcrd/chaincfg"
+	"github.com/decred/politeia/politeiawww/database"
+	"github.com/decred/politeia/politeiawww/sharedconfig"
+	"github.com/decred/politeia/util"
+	"github.com/decred/politeia/util/version"
+)
+
+const (
+	defaultHomeDirname    = "dbutil"
+	defaultDataDirName    = "data"
+	defaultConfigFilename = "politeiawww_dbutil.conf"
+
+	LevelDBOption     = "leveldb"
+	CockroachDBOption = "cockroachdb"
+)
+
+var (
+	defaultHomeDir    = filepath.Join(sharedconfig.DefaultHomeDir, defaultHomeDirname)
+	defaultDataDir    = filepath.Join(sharedconfig.DefaultHomeDir, defaultDataDirName)
+	defaultConfigFile = filepath.Join(defaultHomeDir, defaultConfigFilename)
+)
+
+type Config struct {
+	HomeDir       string `long:"appdata" description:"Path to application home directory"`
+	TestNet       bool   `long:"testnet" description:"Use the test network"`
+	Database      string `long:"database" description:"Database to be used. Valid options are {cockroachdb, leveldb}"`
+	DataDir       string `long:"datadir" description:"Directory where the database is stored"`
+	DBKeyFilename string `long:"dbkeyfilename" description:"File containing the secret key for the database"`
+	DBHost        string `long:"dbhost" description:"Database ip:port"`
+	DBCertDir     string `long:"dbcertdir" description:"Directory containing SSL client certificates"`
+	DBRootCert    string `long:"dbrootcert" description:"File containing SSL root certificate"`
+	EncryptDB     bool   `long:"encryptdb" description:"If true the database will encrypt/decrypt for saving and retrieving records"`
+	RawJSON       bool   `short:"j" long:"json" description:"Print raw JSON output"`
```

Comment:
Why do you have `RawJSON` and `Verbose` as config params when they're not actually used anywhere in the tool?